proxy/localhost.proxy.json

```diff
@@ -8,7 +8,7 @@
       "/apis": "/apis"
     },
     "headers": {
```

Comment:
this file should not be committed, please revert the changes , than it will not come in PR